.prettierrc.json

```diff
@@ -0,0 +1 @@
+{ "tabWidth": 3 }
```

Comment:
[Prettier](https://prettier.io/) helps keep your code base looking neat. It helps with styling. This config tells prettier how to style your project